### ISSUE TITLE
Tx history is not showing a reward lock marker

### ISSUE THREAD

**Issue by msbrogli** [HathorNetwork/hathor-wallet-mobile#43]
When a new block is found, its reward is locked for the next 300 blocks. During this period, the wallet correctly shows 0.00 HTR available and 64.00 HTR locked. But it does not show that the transactions are locked in the history.

Here is attached two screenshots. Notice that the user experience is not good when it is showing 0.00 HTR available with items in the tx history.

![WhatsApp Image 2020-02-13 at 12 51 42](https://user-images.githubusercontent.com/449119/74447880-cd7f9300-4e58-11ea-87d6-ed0ec686e734.jpeg)
![WhatsApp Image 2020-02-13 at 12 51 43](https://user-images.githubusercontent.com/449119/74447884-ceb0c000-4e58-11ea-981f-a9a3e06dc4a2.jpeg)

**Comment by pedroferreira1** [comment]
If we decide to change the UI to show this information we must do it also in the desktop wallet.

Besides that, we should think about all timelock outputs. Should we also show their locked information? Or only from the reward lock? We might have a transaction with 2 outputs, one locked and one not locked, should we show this information on the list?

My opinion is that we shouldn't change the UI to show it. Maybe just make it more clear the locked balance on the mobile wallet, which is hidden by default.

**Comment by obiyankenobi** [comment]
I agree with @pedroferreira1. There are other lock scenarios that would not make it good to have this lock marker. Unless we have a specific one for reward lock. We might have a different icon for it, like we should/could have a different icon for blocks.
